### ISSUE TITLE
cropped line in chart

### ISSUE THREAD

**Issue by dated** [ArkEcosystem/explorer#236]
the line of the chart gets cropped at its maximum value, if the value happens to be the same value of the highest tick (i suppose the same happens if the minimum value is the same of the lowest tick):

![image](https://user-images.githubusercontent.com/6547002/40754316-06672a6e-6479-11e8-8c43-cd85ea099bf7.png)

#159 / #160 were an attempt to fix that, but that static approach fails especially for cryptocurrencies. unfortunately i didn't find a working solution yet.

my last approach was to add a percentage of the difference between the lowest and highest value represented in the chart to the y axis, but that did not work as expected.

```
let minY = Math.min(...this.chartData.datasets[0].data)
let maxY = Math.max(...this.chartData.datasets[0].data)
let delta = maxY - minY

this.options.scales.yAxes[0].ticks.suggestedMax = maxY + delta * 0.05;
```

**Comment by faustbrian** [comment]
Haven't checked the options of chart.js in ages but if I remember right you should be able to simply increase the top padding here.

https://github.com/ArkEcosystem/ark-explorer/blob/master/src/components/ChartWrapper.vue#L41

This should add a top padding inside the canvas and achieve what you want to do. If all 4 values would be 50 you could just do `padding: 50` instead of defining all 4 explicitly.

**Comment by dated** [comment]
unfortunately adding a top padding does not fix this.

**Comment by reConNico** [comment]
This PR was fixed it in the official chartjs repo 3 days ago: https://github.com/chartjs/Chart.js/pull/5321

Integrated the fix manually and it worked. 

We could integrate it manually or either wait for the next chartjs (vuejs wrapper) update.

**Comment by dated** [comment]
perfect! since this is an edge case and does not break functionality i'd say let's wait for the next release.

**Comment by reConNico** [comment]
Will keep an eye on it.

**Comment by faustbrian** [comment]
@dated can you check if it gets fixed if you use the master branch as dependency?

**Comment by dated** [comment]
[Chart.js](https://github.com/chartjs/Chart.js/) is still on v2.7.2 unfortunately. This bugfix won't be available until v2.7.3
